contrib/opentimelineio_contrib/adapters/tests/test_rvsession.py

```diff
@@ -503,16 +508,15 @@ def test_transition_rvsession_read(self):
     def test_image_sequence_example(self):
         # SETUP
         timeline = otio.adapters.read_from_file(IMAGE_SEQUENCE_EXAMPLE_PATH)
-        tmp_path = tempfile.mkstemp(suffix=".rv", text=True)[1]
 
         # EXERCISE
-        otio.adapters.write_to_file(timeline, tmp_path)
+        otio.adapters.write_to_file(timeline, self.tmp_path)
 
         # VERIFY
-        self.assertTrue(os.path.exists(tmp_path))
-        self.assertTrue(os.path.getsize(tmp_path) > 0)
+        self.assertTrue(os.path.exists(self.tmp_path))
+        self.assertTrue(os.path.getsize(self.tmp_path) > 0)
```

Comment:
Since you're checking the content below, this zero size assertion isn't necessary.